app/uk/gov/hmrc/economiccrimelevyregistration/controllers/CheckYourAnswersController.scala

```diff
@@ -107,29 +105,43 @@ class CheckYourAnswersController @Inject() (
           eclDetails(),
           organisationDetails(),
           contactDetails(),
+          otherEntityDetails(),
+          request.registration.otherEntityJourneyData.isDefined(),
```

Comment:
We do not need this Boolean, because you can check previous parameter in the view. More details on view comment